app/Http/Controllers/Task/TaskController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers\Task;
+use App\Task;
+use Illuminate\Http\Request;
+use App\Http\Requests\TaskRequest;
+use App\Http\Controllers\Controller;
+use DB;
+class TaskController extends Controller
+{
+	public function __construct(Task $mTask) {
+		$this->mTask = $mTask;
+	}
+    public function add()
+    {
+    	$Task = DB::table('tasks')->orderBy('id','DESC')->get();
+    	return view('task',compact('Task'));	
+    }
+    public function postAdd(TaskRequest $request)
+    {
+    	$arItem=[
+    		'name'=>trim($request->name),
+    	];
+    	$result = $this->mTask->addItem($arItem);
+    	// $task=new task;
+    	// $task->name = $request->name;
+    	// $result=$task->save(); 
+    	if($result)
+    	{
+    		$request->session()->flash('msg','them thanh cong');
+    	}else{
+    		$request->session()->flash('msg','them that bai');
+    	}
+    	return redirect()->route('task');
+    }
+    public function del($id,Request $request)
```

Comment:
`Request $request` ở đây dùng làm gì thế em

Reply:
dạ Request $request ở đây e dùng để in ra câu thông báo "đã xóa thành công á a "
$request->session()->flash('msgs', 'xóa thành công');